sdk/appconfiguration/app-configuration/package.json

```
@@ -16,15 +16,23 @@
     "isomorphic"
   ],
   "license": "MIT",
-  "main": "./dist/app-configuration.js",
-  "module": "./esm/index.js",
-  "types": "./esm/index.d.ts",
+  "main": "./dist/index.js",
+  "module": "./dist-esm/src/index.js",
+  "types": "./types/src/index.d.ts",
   "devDependencies": {
-    "rollup": "^0.66.2",
-    "rollup-plugin-node-resolve": "^3.4.0",
+    "@types/dotenv": "^6.1.0",
+    "@types/mocha": "^5.2.5",
+    "assert": "^1.4.1",
+    "dotenv": "^8.0.0",
+    "mocha": "^5.2.0",
+    "mocha-junit-reporter": "^1.18.0",
+    "mocha-multi": "^1.0.1",
```

```suggestion
    "mocha-multi": "1.1.0",
```

You need to pin `mocha-multi` to `1.1.0` until we have updated to `mocha@6`.  See https://github.com/Azure/azure-sdk-for-js/pull/4806.